api/src/opentrons/simulate.py

```diff
@@ -440,14 +440,22 @@ def simulate(
     Each dict element in the run log has the following keys:
 
         - ``level``: The depth at which this command is nested - if this an
```

Comment:
space hyphen space, le sigh. 
```suggestion
        - ``level``: The depth at which this command is nested. If this an
```